js/main.js

```diff
@@ -0,0 +1,64 @@
+
+'use strict'
+
+{
+
+  const contents = document.getElementById('text');
+  const addButton = document.getElementById('add');
+  const taskValue = document.getElementById('text');
+  const table = document.getElementById('table');
+  console.log(table);
+  const tbody = document.createElement('tbody');
+
+  table.appendChild(tbody);
+
+  const items = [];
+
+  const showArray = items => {
+
+    tbody.textContent = '';
+
+    const result = items.map((item, i) => {
```

Comment:
ここで利用すべきなのは `map` ではなく `forEach` かなと思います。
```suggestion
    items.forEach((item, i) => {
```